### ISSUE TITLE
daily-comic script not posting complete URL for dilbert

### ISSUE THREAD

**Issue by ScrumpyJack** [sonata82/hubot-daily-comic#6]
when triggering the daily-comic script, the returned URL is missing the protocol
```
//assets.amuniversal.com/67cacba0bdd6013662ea005056a9545d
```

it should be

```
https://assets.amuniversal.com/67cacba0bdd6013662ea005056a9545d
```

xkcd works fine.

+ daily-comic.js@0.0.5
+ hubot-daily-comic@0.0.2

**Comment by sonata82** [comment]
Yes, it seems that the website now uses protocol relative URLs. I thought about fixing this here, but I think daily-comic.js should return a proper URL.

Since I'd like to promote using HTTPS, I'm thinking of changing daily-comic.js to prefix the URL with `https:` if no protocol is given.

**Comment by ScrumpyJack** [comment]
Have you had a chance to look into adding HTTPS to the URL?

**Comment by sonata82** [comment]
Fixed in 0.0.4, now available from [npm](https://www.npmjs.com/package/hubot-daily-comic).
